tdrs-backend/tdpservice/parsers/schema_defs/tribal_tanf/t3.py

```diff
@@ -7,13 +7,14 @@
 from ... import validators
 from tdpservice.search_indexes.documents.tribal import Tribal_TANF_T3DataSubmissionDocument
 
+FIRST_CHILD = 1
+SECOND_CHILD = 2
 
 child_one = RowSchema(
     record_type="T3",
     document=Tribal_TANF_T3DataSubmissionDocument(),
     preparsing_validators=[
-        validators.notEmpty(start=19, end=60),
-        validators.caseNumberNotEmpty(8, 19),
```

Comment:
I think this was accidentally removed?

Reply:
I don't think thats true. That validator checks if the the entire range is empty. If that range has one valid character in it the validator will pass right? Also, even if it did fail the user doesn't know that their case number is malformed.

Reply:
you are correct, but my discussion with Alex on this was that she wanted these to be parsed if and only if at least one of the fields are filled (doesn't mater which). So even if the case_number is empty but another field is filled, then parse. Having both these validators will stop this.
But I also see the point: if the case_number is empty, maybe we have to validate that separately. @ADPennington : what do you think?

Reply:
The notEmpty condition is handled in ```validators.t3_m3_child_validator```